ansible/configs/satellite-vm/files/repos_template.j2

```diff
@@ -8,69 +8,44 @@ enabled=1
 gpgcheck=0
 #gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
 
+{% endif %}
```

Comment:
This file is not used at all in current deploy. Everything should go through the `labsat.opentlc.com` repos